civihr_employee_portal/civihr_employee_portal.module

```diff
@@ -7196,14 +7196,15 @@ function civihr_employee_portal_delete_report_configuration_json($reportName, $c
   drupal_exit();
 }
 
-function civihr_employee_portal_page_alter(&$pages) {
-  $favicon = drupal_get_path('module', 'civihr_employee_portal') . "/images/fav.png";
-  $type = theme_get_setting('favicon_mimetype');
-  drupal_add_html_head_link(array('rel' => 'shortcut icon', 'href' => drupal_strip_dangerous_protocols($favicon), 'type' => $type));
-}
-
+/**
+ * Changes the favicon for both the SSP and CiviHR admin
```

Comment:
Change to 
/**
 * Implements hook_html_head_alter().
 *  Changes the favicon for both the SSP and CiviHR admin.
 */